bootstrap/v2/pkg/kfapp/existing_arrikto/existing.go

```diff
@@ -174,6 +184,14 @@ func (existing *Existing) Apply(resources kftypes.ResourceEnum) error {
 		return internalError(errors.WithStack(err))
 	}
 
+	//for dex the AuthServiceClientSecret shouldn't be base64 encoded as in configMap
```

Comment:
nit: It'd be great to keep the comment formatting homogeneous in the code.

//for -> // For
First letter capital and indented by 1 space.